### PR TITLE
Make GitHub status API context congruous

### PR DESCRIPTION
**Pull request by frapposelli**
Tiny change to make the status api message congruous with all the other CI services out there.

### CODE REVIEW

File: remote/github/github.go [hunk 1]
```diff
@@ -230,7 +230,7 @@ func (g *Github) Status(u *model.User, r *model.Repo, b *model.Build, link strin
 	status := getStatus(b.Status)
 	desc := getDesc(b.Status)
 	data := github.RepoStatus{
-		Context:     github.String("Drone"),
+		Context:     github.String("continuous-integration/drone"),
```
Comment (tboerger) [inline remote/github/github.go]: Maybe even `continuous-integration/drone/pr` like `continuous-integration/travis-ci/pr`?

Comment (frapposelli) [inline remote/github/github.go]: Travis makes a distinction between pull requests and push events, hence the need for a 3rd level, to me they look redundant.

**Comment by bradrydzewski** [comment]
LGTM, will merge once the CLA is signed. Thanks!

**Comment by frapposelli** [comment]
thanks, will do as soon as I hear back from our legal team :sob:

**Comment by frapposelli** [comment]
@bradrydzewski can you get in touch with me over email?

**Comment by bradrydzewski** [comment]
sure, you can email brad@drone.io

**Comment by frapposelli** [comment]
:tada:

**Pull request merged**